### PR TITLE
Feature#15/update landing page for mobile

### LINKED ISSUE #15: Update landing page for mobile
# REQUIREMENTS
Have a mobile view for better interaction

# DESIGN
Captura de Pantalla 2022-04-25 a la(s) 15 45 38

### PR DESCRIPTION
**Pull request by Frcomparan**
# Description
What did you implemented/Why did you implemented this:
## What?
- Updated the landing page design
- Updated the view with mobile first design
- Added menu components to mobile and desktop views  
## Why?
- To give a better style to the page
- To use best practices in the project
- To have a reusable components for the page
  ## Screenshots 
### Previous design 
image
### Mobile
Captura de pantalla 2022-04-26 - 19 09 18
### Desktop
image
### Menus
image
image

image
image

### CODE REVIEW

File: app/assets/stylesheets/abstracts/_classes.scss [hunk 1]
```diff
@@ -7,41 +7,42 @@
...
-  background-color: $light-grey-opacity;
+  background-color: $white;
   border-radius: 20px;
   border: none;
   font-size: 20px;
   font-weight: 400;
   height: 4.5rem;
-  width: 42%;
+  width: 100%;
+  box-shadow: $medium-shadow;
```
Comment (dannharo) [inline app/assets/stylesheets/abstracts/_classes.scss]: sort these properties

File: app/assets/stylesheets/abstracts/_classes.scss [hunk 2]
```diff
@@ -7,41 +7,42 @@
...
-    width: 30%;
+    width: 100%;
 
     label {
       color: black;
       font-weight: 600;
     }
 
     input {
+      width: 100%;
```
Comment (dannharo) [inline app/assets/stylesheets/abstracts/_classes.scss]: sort these properties

File: app/assets/stylesheets/components/_footer.scss [hunk 3]
```diff
@@ -76,44 +78,127 @@ footer {
...
+      text-align: center;
+      display: block;
+    }
+  }
+}
 
-          i{
-            color: $text-grey;          
-          }
+@media (min-width: $breakpoint-mobile) {
```
Comment (dannharo) [inline app/assets/stylesheets/components/_footer.scss:114]: Sort all your css properties, in this file

Comment (Frcomparan) [reply app/assets/stylesheets/components/_footer.scss:114]: Sorted

File: app/assets/stylesheets/abstracts/_classes.scss (hunk 2 above)
Comment (Frcomparan) [reply app/assets/stylesheets/abstracts/_classes.scss]: Sorted

File: app/assets/stylesheets/abstracts/_classes.scss (hunk 1 above)
Comment (Frcomparan) [reply app/assets/stylesheets/abstracts/_classes.scss]: Sorted

**Review by dannharo — APPROVED** [review]

**Review by germanruzca — APPROVED** [review]
LGTM

**Pull request merged**